_includes/lifecycle-status.html

```diff
@@ -0,0 +1,165 @@
+{% if include.data == 'prototype' %}
+  {% assign component_status_items = site.data.lifecycle-status-prototype.components %}
+  {% else %}
+    {% assign component_status_items = site.data.lifecycle-status.components %}
+{% endif %}
+
+{% assign check_icon_classes = 'text-gray-60' %}
+{% assign check_icon_styles = 'height:1rem; width:1rem' %}
+
+{% assign colClasses = '' %}
+{% assign phase_complete_classes = 'bg-gray-5' %}
+{% assign proposal_classes = "lifecycle-bg--proposal" %}
+{% assign accepted_classes = "lifecycle-bg--accepted" %}
+{% assign rejected_classes = "lifecycle-bg--rejected" %}
+{% assign development_classes = "lifecycle-bg--development" %}
+{% assign released_classes = "lifecycle-bg--released" %}
+{% assign experimental_classes = "lifecycle-bg--experimental" %}
+{% assign caution_classes = "lifecycle-bg--caution" %}
+{% assign deprecated_classes = "lifecycle-bg--deprecated" %}
+
+{% assign col1Title = 'Component name' %}
+
+{% assign major_status_types = "proposal, development, released, deprecated" | split: ', '%}
+This table illustrates where USWDS components currently fall in [component lifecycle page]({{ site.baseurl/components/lifecycle }}).
+For a full list of components under consideration for USWDS,
+check out our [component proposals discussion board](http://www.github.com).
+
+Don’t see the component you are looking for? [Start a discussion about it](http://www.github.com).
+
+
+<!-- Start mobile-only status table -->
+<table class="lifecycle-table tablet:display-none">
+  <tbody>
+  {% for item in component_status_items %}
+    {% if item.status_major == "proposal" %}
+      {% if item.status_minor == "will not pursue" %}
+        {% assign status_classes = rejected_classes %}
+        {% elsif item.status_minor == "accepted" %}
+          {% assign status_classes = accepted_classes %}
+        {% else %}
+          {% assign status_classes = proposal_classes %}
+      {% endif %}
+    {% elsif item.status_major == "development" %}
```

Comment:
Consider setting this as the default to make simplify logic.

Are these all the possible states?

```
Proposal (Major)
  Minor - Will not pursue
  Minor - Accepted
  Minor - Other
Development (Major)
  Minor - n/a
Released (Major)
  Minor - Experimental
  Minor - Caution
  Minor - Other
Deprecated (Major)
  Minor - n/a
```

It might be more simple to set a default for each phase and override based on a special case.

**Example**

```
Released (Major)
	Default class
	
	// Overrides
	if Experimental - Experimental
  else if Minor - Caution
```

Or maybe avoid checks altogether and set the status as the class.
```
class="lifecycle-phase--{{ item.status_major }} lifecycle-subphase--{{ item.status_minor }}"
```